tardis/plasma/properties/nlte.py

```diff
@@ -14,4 +21,44 @@ class PreviousElectronDensities(PreviousIterationProperty):
     outputs = ('previous_electron_densities',)
 
 class PreviousBetaSobolevs(PreviousIterationProperty):
-    outputs = ('previous_beta_sobolevs',)
\ No newline at end of file
+    outputs = ('previous_beta_sobolevs',)
+
+class HeliumNLTE(ProcessingPlasmaProperty):
+    outputs = ('helium_population',)
+
+    def calculate(self, level_boltzmann_factor, electron_densities,
+        ionization_data, beta_rad, g, g_electron, w, t_rad, t_electrons,
+        delta, zeta_data, number_density):
+        helium_population = deepcopy(level_boltzmann_factor.ix[2])
+        # He I excited states
+        he_one_population = level_boltzmann_factor.ix[2].ix[0] * \
```

Comment:
@wkerzendorf So the He I level population calculation here is different to what the code normally uses because it's calculated from the GS of the He II ion rather than the He I ion. You're right with the He II and He III level populations though, but it's easier for me to keep all of these calculations in the same function so that I can normalise the populations easily using the number density. But I'm having a think now about ways to simplify it.